core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -378,4 +391,48 @@ public void testSpatialId() {
         assertEquals(5, encoder.getMaxspeed(e3, -1, false), .1);
         assertEquals(-1, encoder.getMaxspeed(e4, -1, false), .1);
     }
+
+    /**
+     * Defines the default rules for German roads
+     *
+     * @author Robin Boldt
+     */
+    private static class TestRule extends DefaultSpatialRule {
```

Comment:
I think we can cleanup this implementation a lot, or just use the GermanSpatialRule again?